pkg/demo/explainer/README.md

```diff
@@ -407,11 +407,11 @@ onChange({eventlet: {value}}, state) {
 
 This will modify the `state.searchFilter` field when the user updates the
 textfield, triggering the particle to be re-rendered. We can update
-the `decorator` and `filter` function to use this new state field.
+the `myDecorator` and `filter` function to use this new state field.
```

Comment:
Maybe we should we update `filter` to `searchFilter` as well?

Reply:
Done